public/stylesheets/style.css

```diff
@@ -95,25 +101,37 @@ img {
 .activity-form{
   color: #a4a4a4;
 }
-.itinerary-list-container-body{
-
+.itinerary-list-container-body {
+    margin-bottom: 10px;
     display: flex;
     flex-direction: row;
     justify-content: space-between;
     align-items: center;
-    border: 1px solid #ddd;
+    border: 5px solid #ddd;
     border-top-left-radius: 4px;
     border-top-right-radius: 4px;
-  }
+}
   .list-group-item:first-child {
     border: none;
   }
 
+  a.list-group-item:focus, a.list-group-item:hover, button.list-group-item:focus, button.list-group-item:hover {
+      color: #555;
+      text-decoration: none;
+      background-color: white;
+  }
 
+  .itinerary-delete{
+    margin-top: 5px;
+  }
   /* ======   UI Button for Sign In TO SCROLL INDEPENDENTLY */
 
   .style {
       background-color: transparent;
       border: none;
       color: #ffffffa6;
   }
+
+  .myclaas{
```

Comment:
myclaas is misspelled. myclass is not a good class name because there is no indication of what it is actually used for. You can look at the previous class names that you added above to get a good idea of what a class name should look like.